### ISSUE TITLE
'W32/Exploit.gen' antivirus alert for latest Windows binary

### ISSUE THREAD

**Issue by EddiesTech** [jackaudio/jack2#656]
According to Panda Antivirus, JACK2 has a 'W32/Exploit.gen' virus on the latest Windows binary. What's going on?

**Comment by falkTX** [comment]
your guess is as good as mine.

looking online, this seems like a generic "alert" that is actually a false positive.
likely will be solved once we sign the release binaries.

**Comment by EddiesTech** [comment]
> your guess is as good as mine.
> 
> looking online, this seems like a generic "alert" that is actually a false positive.
> likely will be solved once we sign the release binaries.

All safe though right?

**Comment by falkTX** [comment]
Yes, the build process is even automated. So unless someone high-jacks travis.. but we would have bigger problems if that happens
